catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/BoundingBoxReader.java

```diff
@@ -111,17 +129,19 @@ public String getWkt() throws CswException {
                     "BoundingBoxReader.getWkt(): could not find either LowerCorner or UpperCorner tags.");
         }
 
-        // Move position back up to the parent <BoundingBox> tag, where we
-        // started
+        // Move position back up to the parent <BoundingBox> tag, where we started
         reader.moveUp();
 
-        LOGGER.debug("Returning WKT in LON/LAT coord order: {}.", wkt);
+        if (!isEPSG4326(crs) || (cswAxisOrder.equals(CswAxisOrder.LAT_LON) && isEPSG4326(crs))) {
```

Comment:
Isn't the second `isEPSG4326(crs)` redundant? If it is FALSE, the first half of the OR is TRUE. If it is TRUE, then the second half of the OR evaluates to 'x && TRUE' which is 'x'.